config/locales/user_mailer/en.yml

```diff
@@ -102,3 +102,8 @@ en:
         you can ignore this message.
       link_text: Go to %{app}
       reset_password: If you can't remember your password, go to %{app} to reset it.
+    undeliverable_address:
+      call_to_action: Please sign in to login.gov and follow instructions.
```

Comment:
We tell people to sign in and follow instructions, but the instructions they see will them to enter the code we couldn't send them, right? Should we build in something, either here or in another PR, that shows users something if their letter bounced?

Reply:
The GUI and this email verbiage is up in the air right now.  I guess depending on where we go with the GUI will drive what we say here.  Docauth currently does not have a way to change the address so we would need to add a new flow for that.

Reply:
Okay, cool. Those are all things we can handle outside this PR once we know what needs to happen.